python/py-contextvars/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-contextvars
+version             2.4
+categories-append   devel
+platforms           darwin
+license             Apache-2
+
+python.versions     36
+
+maintainers         @jandemter openmaintainer
+
+description         backport of Python 3.7 contextvars
+long_description    This package implements a backport of the Python 3.7 \
+                    contextvars module (see PEP 567) for Python 3.6.
+
+homepage            https://github.com/MagicStack/contextvars
+master_sites        pypi:c/contextvars
+
+distname            contextvars-${version}
+
+checksums           rmd160  8a93b02e1ca00c938a16ca579ac32801dd4e4cfb \
+                    sha256  f38c908aaa59c14335eeea12abea5f443646216c4e29380d7bf34d2018e2c39e \
+                    size    9570
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+    depends_lib-append      port:py${python.version}-immutables
+    depends_test-append     port:py${python.version}-pytest
+
+    set pyver [join [split ${python.version} {}] {.}]
```

Comment:
this shouldn't be necessary, to get the correct ```py.test``` version below, you can use the variable ```${python.branch}``` that is defined in the Python PortGroup as in ```test.cmd        py.test-${python.branch}```